larq_compute_engine/mlir/transforms/op_removal_patterns.td

```diff
@@ -22,18 +22,31 @@ def : Pat<(TF_PlaceholderWithDefaultOp $arg), (TF_IdentityOp $arg)>;
 def : Pat<(TF_IdentityOp $arg), (replaceWithValue $arg)>;
 def : Pat<(TF_IdentityNOp $arg), (replaceWithValue $arg)>;
 
-
 // Cleanup dead ops manually. LCE ops are not registered to the TF dialect so
 // op->hasNoSideEffect() will return false. Therefore
 // applyPatternsAndFoldGreedily won't automatically remove the dead nodes. See
 // https://github.com/llvm/llvm-project/blob/master/mlir/include/mlir/IR/Operation.h#L457-L462
 def : Pat<(TF_LceBsignOp:$op $x), (replaceWithValue $x), [(HasNoUseOf:$op)]>;
-def : Pat<(TF_LceBconv2dOp:$op $input, $filter, $post_activation_multiplier,
-            $post_activation_bias, $channels_in,
-            $dilation_height_factor, $dilation_width_factor,
-            $fused_activation_function, $pad_values, $padding,
-            $stride_height, $stride_width),
+def : Pat<(TF_LceBconv2dOp:$op
+              $input,
+              $filter,
+              $post_activation_multiplier,
```

Comment:
I should add that this PR also standardises all the table-gen patterns to use one line per attribute, which should hopefully make future rebases/additions easier.